app/views/admin/submissions/flag.js.erb

```diff
@@ -0,0 +1 @@
+$("#flag-submission-<%= @submission.id %>").parent().html("<%= j (render partial: 'flag', :locals => {submission: @submission}) %>")
```

Comment:
can the element be targeted by submission id, then within the submission, find the flag td/element?

Reply:
Sure, I can do that

Reply:
@ryanwoldatwork  Done